README.md

```diff
@@ -10,37 +10,14 @@ npm install --save mediasuitenz/trimperson#v1.0.0
 ## Usage
 ### setup module:
 ```js
-var trim = require('trim')('http://path/to/trim/api', 'trim-api-security-token')
+var trim = require('trim')({
+  url: 'http://path/to/trim/api',
+  token: 'trim-api-security-token'
+})
```

Comment:
Just to be clear, that wasn't changed in this PR. I noticed that the README was out of date so I fixed it